cli/cmd/root.go

```diff
@@ -133,6 +133,8 @@ func NewCmdRoot() *cobra.Command {
 	f := cmd.PersistentFlags().VarPF(telemetryLevel, "telemetry-level", "", "Telemetry level (none, errors, stats, all)")
 	f.DefValue = "all"
 
+	iUUID := cmd.PersistentFlags().VarPF(&invocationUUID, "invocation-uuid", "", "useful for when using Open Telemetry integration for tracing and logging to be able to correlate logs and traces through many services")
```

Comment:
```suggestion
	iUUID := cmd.PersistentFlags().VarPF(&invocationUUID, "invocation-id", "", "useful for when using Open Telemetry integration for tracing and logging to be able to correlate logs and traces through many services")
```

Let's call the flag by the same name as the log field? The flag is for **Invocation Id**, and the type of the value is **UUID**